src/classes/RD2_ScheduleService.cls

```diff
@@ -364,6 +529,53 @@ public without sharing class RD2_ScheduleService {
         return schedulesByRdId;
     }
 
+    /***
+    * @description Retrieves recurring donation schedules associated with a single recurring donation
+    * @param rd npe03__Recurring_Donation__c
+    */
+    public List<RecurringDonationSchedule__c> getExistingSchedules(npe03__Recurring_Donation__c rd) {
```

Comment:
Do these methods need to be public?

Reply:
The overload for the single RD is actually not used at all.  The list version needs to be public.  I'm removing the unused overload.